helm/hpcc/templates/esp.yaml

```diff
@@ -100,11 +99,18 @@ spec:
         accessDali: "yes"
         app: {{ $application }}
         helmVersion: 8.3.0-trunk0
+{{- if hasKey $.Values.global "metrics" }}
```

Comment:
This should be based on the metrics block pontentially created on line 38: https://github.com/hpcc-systems/HPCC-Platform/blob/7cf6eff4459add2daed0091b5838eb1e32690747/helm/hpcc/templates/esp.yaml#L38
How to reference generated output from template call?

Reply:
you can use fromYaml to get back the dict from what's generated, e.g:
```
$metrics := {{ fromYaml (include generateMetricsConfig  .) }}
```
you might need a default dict after it, to cope with it not generating anything, e.g.:
```
$metrics := {{ fromYaml (include generateMetricsConfig  .) | default dict }}
```
(would need testing)

Reply:
I'm now referencing the "local" metrics. However, it appears the schema doesn't currently allow metrics configuration outside of global, adding temporary values for illustration...